pkg/types/intoto/v0.0.1/entry.go

```diff
@@ -92,6 +92,14 @@ func (v V001Entry) IndexKeys() ([]string, error) {
 				result = append(result, alg+":"+ds)
 			}
 		}
+		predicate, err := parseSlsaPredicate(v.env.Payload)
+		if err == nil {
+			for _, s := range predicate.Predicate.Materials {
```

Comment:
done re style.


the `in_toto.ProvenanceStatement` always has a `Predicate`. It can be empty. But good catch -- `Materials` could be nil, so added that check.